.changeset/purple-poems-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+AdvancedTable - always set the select checkbox column width to `min-content` so it does not grow when the AdvancedTable is narrower than the container
```

Comment:
[issue?] I think there is a specific syntax to use here for an automation tool that @alex-ju has built

Reply:
```suggestion
`AdvancedTable` - always set the select checkbox column width to `min-content` so it does not grow when the `AdvancedTable` is narrower than the container
```